build_helpers/buildAPIDocs.sh

```diff
@@ -31,7 +31,7 @@ var FILES_TO_READ = [
   },
   {
     path: path.join(PROJECT_ROOT, 'src/FixedDataTableCellDefault.js'),
-    name: 'Cell',
+    name: 'DataCell',
```

Comment:
is the component renamed or it was mistyped here

Reply:
`Cell` is the older and now deprecated component.
`DataCell` is the newer replacement. (#687 )